script.js

```diff
@@ -0,0 +1,39 @@
+// $(document).ready(function()
+
+document.onload(runGame();)
+
+function runGame(){
+  document.turn = "X"
+giveMessage(document.turn + " starts the game.");
+}
+
+//tell user whose turn it is, winner, etc
+//function giveMessage
+function giveMessage(msg){
+  document.getElementbyClass("messages").innerhtml = msg;
```

Comment:
This won't work for a couple reasons...
* There is no `getElementbyClass` method. There is a `getElementsByClassName` method.
* The correct method returns a collection of DOM elements. That means you cannot call `.innerHtml` on it -- it can only be called on a single DOM element.
* `.innerhtml` should be spelled as `.innerHtml`.